packages/desktop-client/src/components/util/LoadComponent.tsx

```diff
@@ -9,7 +9,7 @@ type ProplessComponent = ComponentType<Record<string, never>>;
 type LoadComponentProps<K extends string> = {
   name: K;
   message?: string;
-  importer: () => Promise<{ [key in K]: ProplessComponent }>;
+  importer?: () => Promise<{ [key in K]: ProplessComponent }>;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏can `importer` actually be `undefined`? That seems wrong for this component whose primary purpose is to lazy-load a component. Without an `importer` prop this component is basically useless.

Edit: oh I see.. you're using it as a loading indicator. We should not do that. That's not the intention of this component.

Reply:
Reverted LoadComponent Changes and created new element.